velox/runner/tests/LocalRunnerTest.cpp

```diff
@@ -146,26 +148,29 @@ class LocalRunnerTest : public LocalRunnerTestBase {
 
 TEST_F(LocalRunnerTest, count) {
   auto join = makeJoinPlan();
+  auto rootPool = memory::memoryManager()->addRootPool("count");
```

Comment:
You could provide a method in LocalRunnerTest to create a new pool with a monotonically increasing counter? thanks!